addon/components/one-way-input.js

```diff
@@ -47,9 +47,10 @@ export default Component.extend({
   keyUp(event) { this._interpretKeyEvents(event); },
 
   _interpretKeyEvents(event) {
-    const methodName = this.KEY_EVENTS[event.keyCode];
+    let methodName = this.KEY_EVENTS[event.keyCode];
 
     if (methodName) {
+      this._sanitizedValue = null;
```

Comment:
It might be a bit of a hack, but it is needed so key events can fire. Otherwise, the `_sanitizedValue` is always going to be up to date with the raw value at time of key up